src/components/team-gallery/team-gallery.vue

```diff
@@ -0,0 +1,217 @@
+}>();
+
+function addSquishyNameAnimation(index: number) {
+  const teamGalleryMemberName = ref<HTMLElement | null>(null);
+  teamGalleryMemberName.value = document.querySelectorAll(
+    ".team-gallery-member__name"
+  )[index] as HTMLElement;
+
+  teamGalleryMemberName.value.classList.add("team-gallery-squishy-animation");
+
+  teamGalleryMemberName.value.addEventListener("animationend", () => {
+    teamGalleryMemberName.value?.classList.remove("team-gallery-squishy-animation");
+  });
+}
+</script>
+
+<style>
+:root {
+  --team-gallery-grid-columns: 1;
+}
+
+.team-gallery__list {
+  display: grid;
+  grid-template-columns: repeat(var(--team-gallery-grid-columns), 1fr);
+  gap: var(--spacing-small);
+}
+
+.team-gallery__list-item {
+  position: relative;
+}
+
+.team-gallery__list-item:hover
+.team-gallery-member__image {
+  transform: scale(1.05);
+}
+
+.team-gallery__list-item:hover
+.team-gallery-member__name {
+  animation: squishyMotion 500ms ease-in-out;
+}
+
+.team-gallery-squishy-animation {
+  animation: squishyMotion 500ms ease-in-out;
+}
+
+.team-gallery-member__link::after {
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+}
+
+.team-gallery-member__image-wrapper {
+  overflow: hidden;
+  height: auto;
+}
+
+@media (min-width: 486px) {
+  .team-gallery-member__image-wrapper {
+    height: 280px;
+  }
+}
+
+.team-gallery-member__image {
+  display: block;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  transition: transform 200ms ease-in-out;
+}
+
+.team-gallery-member__details {
+  padding: var(--spacing-small);
+  text-align: center;
+  background-color: var(--white);
+}
+
+@media (min-width: 768px) {
+  .team-gallery__list-item:focus-within .team-gallery-member__image {
+    transform: scale(1.05);
+  }
+}
+
+@media (min-width: 486px) {
+  :root {
+    --team-gallery-grid-columns: 2;
+  }
+
+  .team-gallery__list {
+    grid-template-columns: repeat(var(--team-gallery-grid-columns), 1fr);
+  }
+
+  .team-gallery__list-item:nth-child(2n + 2) {
+    transform: translateY(20px);
+  }
+}
+
+@media (min-width: 768px) {
+  :root {
+    --team-gallery-grid-columns: 3;
+  }
+
+  .team-gallery__list .team-gallery__list-item:nth-child(2n + 2) {
+    transform: translateY(0px);
+  }
+
+  .team-gallery__list .team-gallery__list-item:nth-child(3n + 2) {
+    transform: translateY(50px);
+  }
+}
+
+@media (min-width: 1020px) {
+  :root {
+    --team-gallery-grid-columns: 4;
+  }
+
+  .team-gallery__list .team-gallery__list-item:nth-child(2n + 2) {
+    transform: translateY(0px);
+  }
+
+  .team-gallery__list .team-gallery__list-item:nth-child(3n + 2) {
+    transform: translateY(0px);
+  }
+
+  .team-gallery__list .team-gallery__list-item:nth-child(4n + 2),
+  .team-gallery__list .team-gallery__list-item:nth-child(4n + 4) {
+    transform: translateY(50px);
+  }
+}
+
+@media (min-width: 1200px) {
+  :root {
+    --team-gallery-grid-columns: 6;
+  }
+}
+
+@keyframes squishyMotion {
```

Comment:
We've mostly been using `kebab-case` for keyframe animation names' casing. Could you for consistency also that here?

Funky animation though!

Reply:
👍🏻